native/example_low_light_image_enhancement/nnstreamer_example_low_light_image_enhancement.py

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env python
+
+"""
+@file		nnstreamer_example_low_light_image_enhancement.py
+@date		5 Jan 2022
+@brief		example with TF-Lite model for low light image enhancement
+@see		https://github.com/nnstreamer/nnstreamer
+@author		Yelin Jeong <yelini.jeong@samsung.com>
+@bug		No known bugs.
+
+Get model by
+$ cd $NNST_ROOT/bin
+$ bash get-model.sh low-light-image-enhancement
+"""
+
+import os
+import sys
+import logging
+import gi
+import numpy as np
+
+gi.require_version('Gst', '1.0')
+from gi.repository import Gst, GObject
+
+from PIL import Image
+
+class NNStreamerExample:
+    """NNStreamer example for low light image enhancement"""
+
+    def __init__(self, argv=None):
+        self.loop = None
+        self.pipeline = None
+        self.running = False
+        self.tflite_model = ''
+        self.image_name = ''
+
+        if not self.tflite_init():
+            raise Exception
+
+        GObject.threads_init()
+        Gst.init(argv)
+
+    def run_example(self):
+        """Init pipeline and run example.
+        :return: None
+        """
+        # main loop
+        self.loop = GObject.MainLoop()
+
+        # init pipeline
+        self.pipeline = Gst.parse_launch(
+            'filesrc location=' + self.image_name + ' ! pngdec ! videoscale ! '
+            'videoconvert ! video/x-raw,width=600,height=400,format=RGB ! tensor_converter ! '
+            'tensor_transform mode=arithmetic option=typecast:float32,add:0,div:255.0 ! '
+            'tensor_filter framework=tensorflow-lite model=lite-model_zero-dce_1.tflite  ! '
```

Comment:
need to change model path?
```suggestion
            'tensor_filter framework=tensorflow-lite model=tflite_img_low_light_image_enhancement/lite-model_zero-dce_1.tflite  ! '
```

Reply:
Is it better to make a new directory for model?
Now, tflite_init function checks if there is model file in the same directory with python code.

Reply:
I mean, you guided download the model using `get-model.sh`.
```
# bash
$ cd $NNST_ROOT/bin
$ ./get-model.sh low-light-image-enhancement
$ export GST_PLUGIN_PATH=$GST_PLUGIN_PATH:$NNST_ROOT/lib/gstreamer-1.0
$ python3 nnstreamer_example_low_light_image_enhancement.py
```

Reply:
I got it!! I think I should change model path
Thank you 👏